packages/suite-desktop/src-electron/electron.ts

```diff
@@ -185,6 +186,23 @@ const init = async () => {
 
     mainWindow.loadURL(src);
 
+    // Window controls
+    ipcMain.on('window/close', () => {
+        mainWindow.close();
+    });
+    ipcMain.on('window/minimize', () => {
+        mainWindow.minimize();
+    });
+    ipcMain.on('window/maximize', () => {
+        mainWindow.maximize();
+    });
+    ipcMain.on('window/unmaximize', () => {
+        mainWindow.unmaximize();
+    });
+    mainWindow.on('resize', () => {
+        mainWindow.webContents.send('window/is-maximized', mainWindow.isMaximized());
```

Comment:
This message should be sent when the application starts. If you start it maximized, the title bar can display the correct status right away.

Reply:
I created a new `client/ready` event, please check if it could stay like this.